src/MainWindow.vala

```diff
@@ -34,6 +35,11 @@ public class Maya.MainWindow : Hdy.ApplicationWindow {
     private uint configure_id;
     private static Gee.MultiMap<string, string> action_accelerators = new Gee.HashMultiMap<string, string> ();
 
+    private Calendar.Widgets.HeaderBar headerbar;
+    private Calendar.Widgets.ConnectivityInfoBar info_bar;
+    private Gtk.Label error_label;
+    private Gtk.InfoBar error_bar;
```

Comment:
I'm not sure why the scope of all of these widgets have changed. It doesn't look like they are now accessed outside of the construct block. This should probably be reverted